gatsby-config.js

```diff
@@ -26,12 +26,21 @@ module.exports = {
             resolve: `gatsby-remark-images`,
             options: {
               maxWidth: 1120,
+              linkImagesToOriginal: false,
             },
           },
           'gatsby-remark-external-links',
           'gatsby-remark-autolink-headers',
           'gatsby-remark-prismjs',
           'gatsby-remark-copy-linked-files',
+          {
+            resolve: 'gatsby-remark-images-medium-zoom',
+            options: {
+              scrollOffset: 80,
+              zIndex: 9999,
+              excludedSelector: '[src$="symfonycasts-player.png"]',
```

Comment:
Yes, yes, this is not an elegant solution, but unfortunately one thing works, because all any attributes (classes, data attributes) are [removed](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-images/src/index.js#L194) from the original markup due gatsby-remark-image plugin.